### ISSUE TITLE
Most of the ingame screen is black on latest Pop OS 21.04 with Nvidia GPU

### ISSUE THREAD

**Issue by yfuerst** [GTAmodding/re3#1244]
**Describe the bug**
I am playing on Pop OS 21.04 with the latest Nvidia drivers. Most of the screen is black except a small part at the top (see attached image). This problem only occurs in fullscreen. In windowed mode the game works as expected.

Any guesses how to fix this?

**Screenshots**
![gta3](https://user-images.githubusercontent.com/20064424/125949723-ca0ec88c-d9f1-4275-83d3-d144e00240e2.png)

**Version**
I compiled the version from the master branch and tried the compiled version with the latest build.

**Comment by erorcun** [comment]
did you try turning off anti-aliasing?

**Comment by yfuerst** [comment]
That actually fixed it. The highest working anti-aliasing is 8x. Any idea why that is?

**Comment by erorcun** [comment]
No, so I will leave this open until that's fixed. Would you like to change issue title?

**Comment by Tomasak** [comment]
i have same issue on windows with opengl build as mentioned here https://github.com/GTAmodding/re3/issues/1009#issuecomment-770242218

**Comment by TheLonerD** [comment]
Same in opengl builds while using >=8x antialiasing.

**Comment by Calinou** [comment]
MSAA above 8× is generally implemented as a combination of SSAA and MSAA in graphics drivers, which causes weird bugs like https://github.com/godotengine/godot/issues/14727.
